guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,673 @@
+{
```

Comment:
<p>Change to:</p><p><code>gis1 = GIS(profile="your_online_profile")</code></p><p>or</p><p><code>gis1 = GIS("home")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>